### ISSUE TITLE
Can't start streams

### ISSUE THREAD

**Issue by asplundj** [emilsvennesson/kodi-cmore#31]
I can browse titles but no streams will start playing.

Here's my kodi debug log 
https://paste.ubuntu.com/p/5rkq2KX8m7/

I'm running the cmore.no and have tried this on windows, android and coreelec. All running kodi 18RC4

**Comment by kjerstadius** [comment]
I ran into the same issue when I installed the plugin from the Kodi add-on repo. As it turns out, that version is close to a year old and uses an outdated stream API which no longer works. It seem this was fixed back in April of 2018, but I don't know why a new version of the plugin wasn't submitted for inclusion in the Kodi add-on repo at that point (or later).

In any case, installing the plugin from git solved my problems. I think the non-obviousness of this issue could've been avoided quite easily if only the plugin version had been incremented in the xml file. As it stands now, because plugin version of the master branch is the same as the plugin in the Kodi repo, on the surface you'd assume they're one and the same, yet they're not.
